docs/react-native-theta-client-1_2.md

```diff
@@ -25,6 +25,112 @@ yarn run android
 | ----------- | ----------| ---------- |
 | ![screenshot1](images/react_native/windows/scrnshotAndroid1.png){ width="250"} | ![screenshot2](Screenshot_1687290803.png){ width="250"}  | ![screenshot3](Screenshot_1687290798.png){ width="250"}|
 
+## Coding Get Options Component
+
+### 1. Create a new `.tsx` file and Import the Necessary Libraries
+
+From the `react` library we utilize `useEffect` and `useState` to help with State Changes and Asynchronous Calls during the API Requests from the RICOH THETA Client SDK.
+
+From the `react-native` library we utilize the `Text` component to show the data as text on screen.
+
+From [`theta-client-react-native`](https://www.npmjs.com/package/theta-client-react-native) library provided by the RICOH SDK we utilize `OptionNameEnum` and `getOptions` to get data from a chosen [list of Options Enumerators available](https://github.com/ricohapi/theta-client/blob/main/react-native/src/theta-repository/options/options.ts).
+
+
+``` javascript
+import React, {useEffect, useState} from 'react';
+import {Text} from 'react-native';
+
+import {
+  OptionNameEnum,
+  getOptions,
+} from 'theta-client-react-native';
+```
+
+### 2. Create a react component
+
+``` javascript
+const GetOptions = ({ navigation }) => {
+
+    // CODE HERE
+
+  return (
+        <Text></Text>
+    );
+};
+```
+
+### 3. Use RICOH SDK to get Enum Options and Export Component
+
+``` javascript
+const GetOptions = ({ navigation }) => {
+  const [optionNames, setOptionNames] = useState([]);
+
+  useEffect(() => {
+    const fetchOptions = async () => {
+      const result = await getOptions([
+        OptionNameEnum.CameraControlSource,
+        OptionNameEnum.CameraMode,
+        OptionNameEnum.PowerSaving,
+        OptionNameEnum.CaptureMode,
```

Comment:
If these are the 4 options that you pick, as examples, please explain that somewhere. There's a long list of options they could include.

Also, please make sure the output image matches the code.

Reply:
I added new commits that are more consistent with the options